pkg/crds/operator/operator.tigera.io_policyrecommendations.yaml

```diff
@@ -0,0 +1,59 @@
+
+---
```

Comment:
we should remove the extra newline and `---` at the top of this file, like we have for the other CRDs.